### PR TITLE
feat(acme): add support for Redis SSL

### PR DESCRIPTION
**Pull request by vm-001**
### Summary

Add support for Redis SSL

Internal ticket FTI-4432

### CODE REVIEW

**Comment by vm-001** [comment]
Target release version 3.1

File: app/_hub/kong-inc/acme/_index.md [hunk 1]
```diff
@@ -507,6 +510,13 @@ own certificate.
 
 ## Changelog
 
+{% if_plugin_version gte:3.1.x %}
+**{{site.base_gateway}}  3.1.x**
+
+* Added the `config.storage_config.ssl`, `config.storage_config.ssl_verify`, and `config.storage_config.ssl_server_name`. configuration parameters.
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/acme/_index.md:516]:
```suggestion
* Added the `config.storage_config.ssl`, `config.storage_config.ssl_verify`, and `config.storage_config.ssl_server_name` configuration parameters.
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
@vm-001 Thanks for submitting this! Since this is for Gateway 3.1, could you change the base branch to `release/gateway-3.1`?

**Comment by vm-001** [comment]
> @vm-001 Thanks for submitting this! Since this is for Gateway 3.1, could you change the base branch to `release/gateway-3.1`?

Done.

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Thanks for submitting this!

**Pull request merged**